Importer/Interpreter/Field/BooleanFieldInterpreter.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace Netdudes\ImporterBundle\Importer\Interpreter\Field;
+
+use Netdudes\ImporterBundle\Importer\Configuration\Field\BooleanFieldConfiguration;
+use Netdudes\ImporterBundle\Importer\Configuration\Field\FieldConfigurationInterface;
+
+class BooleanFieldInterpreter implements FieldInterpreterInterface
+{
+    /**
+     * @var array
+     */
+    private $falseRepresentations = [
+        'false',
+        false,
+        '0',
+        0
+    ];
+
+    /**
+     * @param FieldConfigurationInterface $configuration
+     * @param mixed                       $value
+     *
+     * @return bool|null
+     */
+    public function interpret(FieldConfigurationInterface $configuration, $value)
+    {
+        if (!($configuration instanceof BooleanFieldConfiguration)) {
+            throw new \InvalidArgumentException();
+        }
+
+        if (null === $value) {
+            return null;
+        }
+
+        foreach($this->falseRepresentations as $falseRepresentation){
```

Comment:
There is a boolean parameter `strict` that can be passed as third parameter in the `in_array`. This is `False` by default but If you change it to `True` I think it will work.